src/systems/frequency_system.C

```diff
@@ -339,9 +343,11 @@ void FrequencySystem::solve (const unsigned int n_start,
   // Get the user-specified linear solver tolerance,
   //     the user-specified maximum # of linear solver iterations,
   //     the user-specified wave speed
-  const Real tol            =
+  const Real tol            = parameters.have_parameter<Real>("linear solver tolerance") ?
+    double(parameters.get<Real>("linear solver tolerance")) :
     es.parameters.get<Real>("linear solver tolerance");
-  const unsigned int maxits =
+  const unsigned int maxits = parameters.have_parameter<unsigned int>("linear solver maximum iterations") ?
+    parameters.get<unsigned int>("linear solver maximum iterations") :
     es.parameters.get<unsigned int>("linear solver maximum iterations");
```

Comment:
As long as we have to change all these anyway, wouldn't `auto [maxits, tol] = this->get_linear_solve_parameters();` work?  We can't do that in `EigenSystem` because it doesn't inherit from `ImplicitSystem`, but it seems like it ought to work here, and in `LinearImplicitSystem`, and (albeit for just these two parameters) in `NonlinearImplicitSystem`.